obspy/io/sac/core.py

```diff
@@ -381,7 +381,8 @@ def _internal_read_sac(buf, headonly=False, debug_headers=False, fsize=True,
     :return: A ObsPy Stream object.
     """
     # read SAC file
-    sac = SACTrace.read(buf, headonly=headonly, ascii=False, checksize=fsize)
+    sac = SACTrace.read(buf, headonly=headonly, ascii=False, checksize=fsize,
+                        encoding=kwargs.get('encoding', 'ASCII'))
```

Comment:
@megies @jkmacc-LANL @krischer, I have a quick question for you guys. `_internal_read_sac` is called by the following two function. If we want to add a parameter 'encoding' to it, do I need to add that parameter to these two functions as well or `**kwargs` will take care it for me?

`obspy.core.stream._read`:
```Python
    stream, format = _read_from_plugin('waveform', filename, format=format,
                                       headonly=headonly, **kwargs)
```

`obspy.core.util.base._read_from_plugin`:
```Python
   # read
    list_obj = read_format(filename, **kwargs)
```